web-wallet/src/lib/components/Transactions/Transactions.svelte

```diff
@@ -27,6 +27,12 @@
   /** @type {Promise<Transaction[]>} */
   export let items;
 
+  /** @type {Boolean}*/
+  export let isSyncing;
+
+  /** @type {Error|null}*/
+  export let error;
```

Comment:
This is more of a nitpick, but I would have this as a `syncError` instead, so that it's clear it's not related to an error triggered by the component itself.